### PR TITLE
changing from opencensus to opentelemetry

### PR DESCRIPTION
**Pull request by ardan-bkennedy**

### CODE REVIEW

File: cmd/sales-api/main.go [hunk 1]
```diff
@@ -78,10 +75,9 @@ func run(log *log.Logger) error {
 			Algorithm      string `conf:"default:RS256"`
 		}
 		Zipkin struct {
-			LocalEndpoint string  `conf:"default:0.0.0.0:3000"`
-			ReporterURI   string  `conf:"default:http://zipkin:9411/api/v2/spans"`
-			ServiceName   string  `conf:"default:sales-api"`
-			Probability   float64 `conf:"default:0.05"`
```
Comment (liyongjing) [inline cmd/sales-api/main.go:84]: Zipkin struct remove Probability property ?

Comment (ardan-bkennedy) [reply cmd/sales-api/main.go:84]: I'm passing that into Init because of your comment about not really wanting to sample all.

**Comment by ardan-bkennedy** [comment]
@liyongjing I need your name/email for the CONTRIBUTORS file.

File: cmd/sales-api/main.go (hunk 1 above)
Comment (liyongjing) [reply cmd/sales-api/main.go:84]: Init function not use this property

**Comment by liyongjing** [comment]
@liyongjing  leeyongjing@gmail.com

**Pull request merged**